doc/classes/EditorTranslationParserPlugin.xml

```diff
@@ -106,14 +106,50 @@
 		<method name="_get_recognized_extensions" qualifiers="virtual const">
 			<return type="PackedStringArray" />
 			<description>
-				Gets the list of file extensions to associate with this parser, e.g. [code]["csv"][/code].
+				Override this method to provide the list of file extensions to associate with this parser, e.g. [code]["csv"][/code]. See also [method get_recognized_extensions].
 			</description>
 		</method>
 		<method name="_parse_file" qualifiers="virtual">
 			<return type="PackedStringArray[]" />
 			<param index="0" name="path" type="String" />
 			<description>
-				Override this method to define a custom parsing logic to extract the translatable strings.
+				Override this method to define a custom parsing logic to extract the translatable strings. See also [method parse_file].
+			</description>
+		</method>
+		<method name="get_all_recognized_extensions" qualifiers="static">
+			<return type="PackedStringArray" />
+			<description>
+				Returns a list of all file extensions supported by any translation parser, both standard and custom.
+			</description>
+		</method>
+		<method name="get_builtin_strings" qualifiers="static">
+			<return type="PackedStringArray[]" />
+			<description>
+				Returns a list of strings from built-in components such as certain [Control] nodes in the format [code][msgid, msgctxt, msgid_plural][/code].
```

Comment:
To me, this description was not too clear at first read. Maybe something like "each item in the array is a packedstringarray with the following entries: ..."